scripts/lis_hyperv_platform/main.ps1

```diff
@@ -158,17 +152,28 @@ function Get-Dependencies {
         [string] $xmlTest
     )
     if ( Test-Path $keyPath ){
-        cp "$keyPath" ".\lis-test\WS2012R2\lisa\ssh"
+        cp "$keyPath" "${LISA_REL_PATH}\ssh"
     }
     $keyName = ([System.IO.Path]::GetFileName($keyPath))
     if ( Test-Path $xmlTest ){
-        cp $xmlTest ".\lis-test\WS2012R2\lisa\xml"
+        cp $xmlTest "${LISA_REL_PATH}\xml"
         $xmlName = ([System.IO.Path]::GetFileName($xmlTest))
     } else {
         $xmlName = $xmlTest
     }
     return ($keyName, $xmlName)
 }
+
+function Copy-LisaTestDependencies {
+    param([string[]] $TestDependenciesFolders)
+
+    # This function copies test dependencies in lisa folder from a given share
```

Comment:
throw an error if the folder does not exit: LisaTestDependencies
if (!(Test-Path $LisaTestDependencies)) {
    throw "${LisaTestDependencies} path does not exist!"
}